File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/FoxtrotVosController.java
package nl.rsdt.japp.jotial.maps.management.controllers;


/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
public class FoxtrotVosController extends VosController {

    public static final String CONTROLLER_ID = "FoxtrotVosController";

    public static final String STORAGE_ID = "STORAGE_VOS_F";

    public static final String BUNDLE_ID = "VOS_F";

    public static final String REQUEST_ID = "REQUEST_VOS_F";

    public FoxtrotVosController() {
    }

    @Override
    public String getTeam() {
        return "f";
    }

    @Override
    public String getId() {
        return CONTROLLER_ID;
    }

    @Override
    public String getStorageId() {
        return STORAGE_ID;
    }

    @Override
    public String getBundleId() {
        return BUNDLE_ID;
    }

}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/FoxtrotVosController.kt
package nl.rsdt.japp.jotial.maps.management.controllers


/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
class FoxtrotVosController : VosController() {

    override val team: String
        get() = "f"

    override val id: String
        get() = CONTROLLER_ID

    override val storageId: String
        get() = STORAGE_ID

    override val bundleId: String
        get() = BUNDLE_ID

    companion object {

        val CONTROLLER_ID = "FoxtrotVosController"

        val STORAGE_ID = "STORAGE_VOS_F"

        val BUNDLE_ID = "VOS_F"

        val REQUEST_ID = "REQUEST_VOS_F"
    }

}

